projects/storefrontlib/src/cms-components/user/logout/logout-guard.ts

```diff
@@ -30,14 +52,20 @@ export class LogoutGuard implements CanActivate {
         type: PageType.CONTENT_PAGE,
       })
       .pipe(
-        tap(hasPage => {
-          if (!hasPage) {
```

Comment:
The initial idea of Tobais was to allow customers to implement logout landing page, if they want. By default in our sample data there is no such a 'logout' CMS page, so we fallback to redirecting to other page (home). If you don't mind, you can leave a code comment here with the original purpose - for future reference.

The introduced `filter` doesn't allow for custom `logout` CMS page.

Reply:
Isn't the new solution exactly the same as the previous one, but more elegant? I cannot see a scenario where one would work and another would not. Can u give me such example?

Reply:
`filter(hasPage => !hasPage),` will never emmit a value if the `logout` page exists in CMS.
So the guard will never emmit value.
So you will never navigate to this page.

Reply:
fair enough, reverting